hs/database/Tables.hs

```diff
@@ -77,25 +77,26 @@ Graphs
 
 Rects
     gId Int
-    rId Int
+    rId String
```

Comment:
This has been changed to a String for the time being. In the future, I hope to use the Persistent database primary keys.